examples/step-61/step-61.cc

```diff
@@ -0,0 +1,885 @@
+                            cell_matrix_rt(i,j) += (
+                           		                   phi_i_u * phi_j_u *
+                           		                   fe_values_rt.JxW(q));
+                           }
+                     }
+                }
+
+         // We take the inverse of the Gram matrix, take matrix multiplication and get the matrix with coefficients of projection.
+          cell_matrix_rt.gauss_jordan();
+          cell_matrix_rt.mmult(cell_matrix_D,cell_matrix_E);
+
+         // This is to extract pressure values of the element.
+          cell->get_dof_indices (local_dof_indices);
+          cell_solution = 0;
+          for (unsigned int i=0; i<dofs_per_cell; ++i)
+              {
+             	cell_solution(i) = solution(local_dof_indices[i]);
+              }
+
+           // This cell matrix will be used to calculate the coefficients of the Gram matrix.
+           // This part is the same as the part in evaluating pressure.
+           for (unsigned int q=0; q<n_q_points; ++q)
+               {
+                 for (unsigned int i=0; i<dofs_per_cell; ++i)
+                	 {
+                       for (unsigned int k=0; k<dofs_per_cell_rt; ++k)
+                           {
+                 	          const double phi_k_u_div = fe_values_rt[velocities].divergence(k,q);
+                 	          cell_matrix_F(i,k) -= (fe_values[interior].value(i,q) *
+                 	                                 phi_k_u_div *
+                 	                                 fe_values.JxW (q));
+                 	        }
+                 	  }
+                 }
+
+            for (unsigned int face_n=0; face_n<GeometryInfo<dim>::faces_per_cell; ++face_n)
+                {
+                   fe_face_values.reinit (cell,face_n);
+                   fe_face_values_rt.reinit (cell_rt,face_n);
+                   for (unsigned int q=0; q<n_face_q_points; ++q)
+                       {
+                     	 const Tensor<1,dim> normal = fe_face_values.normal_vector(q);
+                         for (unsigned int i=0; i<dofs_per_cell; ++i)
+                         	  {
+                         	    for (unsigned int k=0; k<dofs_per_cell_rt; ++k)
+                         	        {
+                         	          const Tensor<1,dim> phi_k_u = fe_face_values_rt[velocities].value (k, q);
+                         	          cell_matrix_F(i,k) += (fe_face_values[face].value(i,q) *
+                         	                                (phi_k_u * normal)*
+                         	                                 fe_face_values.JxW (q));
+                         	         }
+                         	   }
+                         }
+                  }
+
+             cell_matrix_F.mmult(cell_matrix_C,cell_matrix_rt);
+
+             // From previous calculations, we have all the coefficients. Then we get the coefficient @p beta.
```

Comment:
`From previous calculations we obtained all the coefficients neede to calculate beta.`